sample_apps/common/example_app.cc

```diff
@@ -55,10 +55,11 @@ DEFINE_string(platform_attest_endorsement,
               "platform endorsement of attest key");
 DEFINE_string(attest_key_file, "attest_key_file.bin", "attest key");
 DEFINE_string(measurement_file, "example_app.measurement", "measurement");
+
 DEFINE_string(public_key_alg, Enc_method_rsa_2048, "public key algorithm");
-DEFINE_string(symmetric_key_alg,
+DEFINE_string(auth_symmetric_key_alg,
```

Comment:
For brevity, the command-line flag to name these algorithms will be `--auth_symmetric_key_alg`